.github/workflows/ci.yaml

```
@@ -11,55 +11,22 @@ env:
   PRE_COMMIT_HOME: ${{ github.workspace }}/.precommit_cache
 
 permissions:
-    id-token: write
-    contents: write # needed for mutex
+  id-token: write # needed to assume OIDC roles (e.g. for downloading from CodeArtifact)
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)